django_ipam/templates/admin/django-ipam/change_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+{% load static %}
+
+{% block after_related_objects %}
+    <li style="list-style-type:none">
+      <div id="graph"></div>
+      {% if total %}
+        <div>
+          <h3>Subnet Visual Display</h3>
+          <ul class="subnet_visual">
+            {% for ipaddr in total %}
+              {% if ipaddr in used_ip %}
+                <li>
+                  <a class="used">
+                    {{ ipaddr }}
+                  </a>
+                </li>
+              {% else %}
+                <li>
+                  <a href="/admin/django_ipam/ipaddress/add/?_to_field=id&amp;_popup=1&amp;ip_address={{ ipaddr }}&amp;subnet={{ subnet.id }}" onclick='return showAddAnotherPopup(this);'>
+                    {{ ipaddr }}
+                  </a>
+                </li>
+              {% endif %}
+            {% endfor %}
+          </ul>
+        </div>
+      {% else %}
+        <div class="list_ips">
+          <h3>Used IP address</h3>
+          <ul class="subnet_visual">
+            {% for ipaddr in used_ip %}
+            <li>
+              <a class="used" href="/admin/django_ipam/ipaddress/{{ ipaddr.id }}/change" onclick='return showAddAnotherPopup(this);'>{{ ipaddr }}</a>
+            </li>
+            {% endfor %}
+          </ul>
+        </div>
+      {% endif %}
+    </li>
+{% endblock %}
+
+{% block admin_change_form_document_ready %}
+  {{ block.super }}
+  <script type="text/javascript" src="{% static 'django-ipam/js/minified/plotly.min.js' %}"></script>
+  <script type="text/javascript">
+    var data = [],
+        layout = {}
+    data = [
+      {
+        values: {{values|safe}},
+        labels: {{labels|safe}},
```

Comment:
as explained previously, please use the format ``{{ labels|safe }}`` (space between brackets)